web_widget_x2many_2d_matrix/README.rst

```diff
@@ -133,11 +133,6 @@ Now in our wizard, we can use:
 Known issues / Roadmap
 ======================
 
-* Support extra attributes on each field cell via `field_extra_attrs` param.
```

Comment:
please update the whole readme. You miss the part "You can pass the following parameters:"

Reply:
Yeah, please include an example of the usage on the README